README.md

```diff
@@ -29,9 +29,11 @@ This will create and fill an index called `taxi`, using the short url list in us
 
 If you want to try out the full data set, run this:
 
-`pdk taxi -d taxi-big -f usecase/taxi/urls.txt`
+`pdk taxi -i taxi-big -f usecase/taxi/greenAndYellowUrls.txt`
 
-Note that this url file represents 1+ billion rows of data.
+There are a number of other options you can tweak to affect the speed and memory usage of the import (or point it to a remote pilosa instance). Use `pdk taxi -help` to see all the options.
+
+Note that this url file represents 1+ billion columns of data. 
```

Comment:
I intended this line as a warning, but it may not read that way to a newcomer. Maybe we should explicitly point out that this means large time/memory/disk requirements.